docs/src/codegen.md

```diff
@@ -156,55 +192,121 @@ playwright codegen --color-scheme=dark playwright.dev
 pwsh bin/Debug/netX/playwright.ps1 codegen --color-scheme=dark playwright.dev
 ```
 
-<img width="1258" alt="Codegen generating code for tests for playwright.dev website in dark mode" src="https://user-images.githubusercontent.com/13063165/182359371-0bb4a7a2-abbb-4f73-8550-d67e0101f0ad.png" />
+######
+* langs: js
+
+<img width="1394" alt="Codegen generating code for tests for playwright.dev website in dark mode js" src="https://user-images.githubusercontent.com/13063165/220412436-5f382f44-7dd7-4169-8acc-cc4894dc513c.png" />
+
+######
+* langs: java
+
+<img width="1394" alt="Codegen generating code for tests for playwright.dev website in dark mode java" src="https://user-images.githubusercontent.com/13063165/220412615-509e9810-2daa-45d9-86ea-efd108be0e51.png" />
+
+######
+* langs: python
+
+<img width="1394" alt="Codegen generating code for tests for playwright.dev website in dark mode python" src="https://user-images.githubusercontent.com/13063165/220412939-a54c1b0b-b758-4fc3-88d2-719f5e885aaf.png" />
+
+######
+* langs: csharp
+
+<img width="1394" alt="Codegen generating code for tests for playwright.dev website in dark mode csharp" src="https://user-images.githubusercontent.com/13063165/220413119-9bc1c77b-2609-4b21-861c-5786b9ab48f9.png" />
 
 ### Emulate geolocation, language and timezone
 
-Record scripts and tests while emulating timezone, language & location using the `--timezone`, `--geolocation` and `--lang` options. Once page opens, click the "show your location" icon at them bottom right corner of the map to see geolocation in action.
+Record scripts and tests while emulating timezone, language & location using the `--timezone`, `--geolocation` and `--lang` options. Once the page opens, click the "show your location" icon on the map to see geolocation in action.
```

Comment:
Since you change Google Maps to Bing Maps, you need to change the instructions here too.
1. Accept the cookies
2. on the top right click on the locate me button.